lib/identity_config.rb

```diff
@@ -153,14 +153,14 @@ def self.build_store(config_map)
     config.add(:disable_email_sending, type: :boolean)
     config.add(:disallow_all_web_crawlers, type: :boolean)
     config.add(:disposable_email_services, type: :json)
+    config.add(:doc_auth_agreement_controller_enabled, type: :boolean)
     config.add(:doc_auth_attempt_window_in_minutes, type: :integer)
     config.add(:doc_auth_client_glare_threshold, type: :integer)
     config.add(:doc_auth_client_sharpness_threshold, type: :integer)
     config.add(:doc_auth_error_dpi_threshold, type: :integer)
     config.add(:doc_auth_error_glare_threshold, type: :integer)
     config.add(:doc_auth_error_sharpness_threshold, type: :integer)
     config.add(:doc_auth_extend_timeout_by_minutes, type: :integer)
-    config.add(:doc_auth_hybrid_handoff_controller_enabled, type: :boolean)
```

Comment:
Is it intentional to remove this? It looks like it's still used in a number of places.

Reply:
Yes, I was anticipating merging in #8510 which I will do as soon as it gets an approval.